### PR TITLE
 Add vector tile examples 

### LINKED ISSUE #31: Add vector layer example(s)
- Add OpenLayers example
- Add Cesium and Leaflet examples (Potentially)

### PR DESCRIPTION
**Pull request by Benjaki2**
Fixes #31

Adds examples of various ways to use vector tiles.
- [x] Example of using vector tiles with Openlayers style classes
- [x] Vector interaction example (hover feature and get data)
- [x] Update Openlayers
- [x] Update readme

Is there anything else that should included here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ source for
   * [Antarctic (EPSG:3031)](https://nasa-gibs.github.io/gibs-web-examples/examples/openlayers/antarctic-epsg3031.html)
   * [Web Mercator (EPSG:3857)](https://nasa-gibs.github.io/gibs-web-examples/examples/openlayers/webmercator-epsg3857.html)
   * [Rolling Seven Day Slider](https://nasa-gibs.github.io/gibs-web-examples/examples/openlayers/time.html)
+  * [Vector Tile Basic Use](https://github.com/nasa-gibs/web-examples/blob/master/examples/openlayers/vectors/geographic-epsg4326-vector-basic.html)
```
Comment (jasontk19) [inline README.md:15]: Should these also be linked from the `index.html` so you can easily get to them when running the examples locally?

Comment (Benjaki2) [reply README.md:15]: Yes, Definitely. Good find.

File: examples/openlayers/vectors/geographic-epsg4326-vector-basic.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<!--
+   GIBS Web Examples
+
+   Copyright 2013 - 2018 United States Government as represented by the
```
Comment (minniewong) [inline examples/openlayers/vectors/geographic-epsg4326-vector-basic.html]: I guess the copyright should be changed to 2020?

Comment (Benjaki2) [reply examples/openlayers/vectors/geographic-epsg4326-vector-basic.html]: Yeah, that makes sense to me. 👍  I think we removed these banners in favor of keeping them in one place for WV a while back. Something to consider here as well... But for now I updated them ©️

**Review by minniewong — APPROVED** [review]
Seems to look good to me!

File: README.md [hunk 3]
```diff
@@ -67,12 +70,15 @@ uses GIBS as its primary image source.
...
 
-This example uses [OpenLayers](http://openlayers.org/) version 5.1.3.
+These examples use [OpenLayers](http://openlayers.org/) version 6.4.3.
 
 If geometry transformations are required using coordinates in the polar systems,
 [proj4js](http://trac.osgeo.org/proj4js), version 2, must be included. This
 example uses proj4js version 2.4.3. This is not required to simply display the map.
 
+#### Mapbox Vector Tile (MVT) Examples
+[Mapbox Vector Tiles](https://docs.mapbox.com/vector-tiles/specification/) are available for some of the GIBS vector datasets. These Openlayers examples show how to style vector tiles using Openlayers styling classes, how to style vector tiles with the [Mapbox Styles](https://docs.mapbox.com/mapbox-gl-js/style-spec/) that are provided by GIBS in the [getCapabilities](https://gibs.earthdata.nasa.gov/wmts/epsg4326/std/wmts.cgi?request=GetCapabilities), and how to vizualize feature data through map interactions.
```
Comment (rboller) [inline README.md]: vizualize --> visualize

**Review by rboller — CHANGES_REQUESTED** [review]
Just the one typo fix from me and otherwise looks good!

**Pull request merged**